packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -93,35 +92,27 @@
 	align-items: center;
 	display: flex;
 	justify-content: center;
-	height: 250px;
+	height: $grid-unit-80 * 4; // 256px
```

Comment:
This is the height of the area where we will upload fonts. This is the only exception, and I adjusted it to a multiple of 8.

![upload-area](https://github.com/WordPress/gutenberg/assets/54422211/ed1551c7-4890-4715-a48b-1d9e7ea9c005)